samples/java/build.gradle

```diff
@@ -7,6 +7,12 @@ repositories {
     mavenCentral()
 }
 
+java {
```

Comment:
If we do this for the Export API sample, we should also do this for the Enterprise API sample.